configs/unet/README.md

```diff
@@ -21,7 +21,7 @@
 
 |  Method   | Backbone      | Image Size | Crop Size | Stride | Lr schd | Mem (GB) | Inf time (fps) |  Dice | config                                                                                                                   | download                                                                                                                                                                                                                                                                                                                                             |
 | ----------- | --------- | ---------- | --------- | -----: | ------- | -------- | -------------: | ----: | ------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-|  FCN |  UNet-S5-D16      | 584x565    | 64x64     |  42x42 | 40000   | 0.680    |              - | 78.67 | [config](https://github.com/open-mmlab/mmsegmentation/blob/master/configs/unet/fcn_unet_s5-d16_64x64_40k_drive.py)       | [model](https://download.openmmlab.com/mmsegmentation/v0.5/unet/unet_s5-d16_64x64_40k_drive/unet_s5-d16_64x64_40k_drive_20201223_191051-5daf6d3b.pth) &#124; [log](https://download.openmmlab.com/mmsegmentation/v0.5/unet/unet_s5-d16_64x64_40k_drive/fcn_unet_s5-d16_64x64_40k_drive-20201223_191051.log.json)                         |
+|  FCN |  UNet-S5-D16      | 584x565    | 64x64     |  42x42 | 40000   | 0.680    |              - | 78.67 | [config](https://github.com/open-mmlab/mmsegmentation/blob/master/configs/unet/fcn_unet_s5-d16_64x64_40k_drive.py)       | [model](https://download.openmmlab.com/mmsegmentation/v0.5/unet/unet_s5-d16_64x64_40k_drive/unet_s5-d16_64x64_40k_drive_20201223_191051-5daf6d3b.pth) &#124; [log](https://download.openmmlab.com/mmsegmentation/v0.5/unet/unet_s5-d16_64x64_40k_drive/unet_s5-d16_64x64_40k_drive-20201223_191051.log.json)                         |
```

Comment:
`fcn_unet` is the valid model name.
So, we need to change filename of checkpoint in oss.